RELEASING.md

```diff
@@ -0,0 +1,58 @@
+# Developer documentation
+
+This document summarizes information relevant to Storm committers.  It includes information about
+the Storm release process.
+
+---
+
+# Release process
+
+## Preparation
+
+Ensure you can log in to http://repository.apache.org. You should use your Apache ID username and password.
+
+Install an svn client, and ensure you can access the https://dist.apache.org/repos/dist/dev/storm/ and https://dist.apache.org/repos/dist/release/storm/ repositories. You should be able to access these with your Apache ID username and password.
+
+Ensure you have a signed GPG key, and that the GPG key is listed in the Storm KEYS file at https://dist.apache.org/repos/dist/release/storm/KEYS. The key should be hooked into the Apache web of trust. You should read the [Apache release signing page](http://www.apache.org/dev/release-signing.html), the [release distribution page](http://www.apache.org/dev/release-distribution.html#sigs-and-sums), as well as the [release publishing](http://www.apache.org/dev/release-publishing) and [release policy](http://www.apache.org/legal/release-policy.html) pages.
+
+## Setting up a vote
+
+1. Run `mvn release:prepare` followed `mvn release:perform` on the branch to be released. This will create all the artifacts that will eventually be available in maven central. This step may seem simple, but a lot can go wrong (mainly flaky tests).
+
+2. Once you get a successful maven release, a “staging repository” will be created at http://repository.apache.org in the “open” state, meaning it is still writable. You will need to close it, making it read-only. You can find more information on this step [here]{www.apache.org/dev/publishing-maven-artifacts.html}.
```

Comment:
[here]{www.apache.org/dev/publishing-maven-artifacts.html}. seems wrong. It needs to be  "()" instead of "{}"

Reply:
Good eyes, fixed